### PR TITLE
update newbie track

### PR DESCRIPTION
**Pull request by dspt**
Let me know if this is too fluffy or not descriptive enough.

### CODE REVIEW

File: content/pages/entry.rst [hunk 1]
```diff
@@ -9,14 +10,23 @@ Here are some of the topics that will be covered:
...
 ---------------------
-We'll go over the basics of Linux, the terminal, irc, and other awesome tools
-that we'll use throughout Devops Bootcamp.
-
-Single System Fundamentals Workshop
------------------------------------
-We'll explore the basics of Linux System fundamentals, including file
-permissions, users, groups, package management and a few other misc related
-topics.
+We'll go over what DevOpsBootcamp is all about and get you started with your
```
Comment (ElijahCaine) [inline content/pages/entry.rst]: DevOps BootCamp is two words.

File: content/pages/entry.rst [hunk 2]
```diff
@@ -9,14 +10,23 @@ Here are some of the topics that will be covered:
...
+own system that you can start following our lessons in. All that's required is
```
Comment (ElijahCaine) [inline content/pages/entry.rst]: Change ``that you can start following our lessons in`` to ``that you can use for hands on demos``, or something like that. Following our lessons in doesn't quite make sense.

File: content/pages/entry.rst [hunk 3]
```diff
@@ -9,14 +10,23 @@ Here are some of the topics that will be covered:
...
+
+Introduction to Linux and the Command Line
+------------------------------------------
+Next we will begin teaching how to interact with your new linux server; the
+command line and basics of linux will be shown through exercises designed to
+give you the confidence to start exploring.
+
+IRC and Getting Help
+--------------------
+IRC is Internet Relay Chat and an integral part of any internet based
```
Comment (ElijahCaine) [inline content/pages/entry.rst]: ``Internet Relay Chat (IRC)``

**Comment by ElijahCaine** [comment]
I like it a lot. Just a few minor tweaks in wording. +1 after those are taken.

File: content/pages/entry.rst [hunk 4]
```diff
@@ -9,14 +10,23 @@ Here are some of the topics that will be covered:
...
-permissions, users, groups, package management and a few other misc related
-topics.
+We'll go over what DevOpsBootcamp is all about and get you started with your
+own system that you can start following our lessons in. All that's required is
+a laptop and web browser.
+
+
+Introduction to Linux and the Command Line
+------------------------------------------
+Next we will begin teaching how to interact with your new linux server; the
```
Comment (lucywyman) [inline content/pages/entry.rst]: s/server/system ?

Comment (ramereth) [inline content/pages/entry.rst]: Not sure if that distinction is really needed. I think using server here is OK.

**Comment by ramereth** [comment]
@dspt you might have @Phillels take a look at the wording.

**Comment by Phillels** [comment]
I wouldn't use 'any' in the paragraph describing IRC. Other than that it looks great.

**Pull request merged**